rmw_connext_shared_cpp/src/node_names.cpp

```diff
@@ -67,15 +71,28 @@ get_node_names(
     return RMW_RET_BAD_ALLOC;
   }
 
-
   for (auto i = 1; i < length; ++i) {
     DDS::ParticipantBuiltinTopicData pbtd;
     auto dds_ret = participant->get_discovered_participant_data(pbtd, handles[i - 1]);
-    const char * name = pbtd.participant_name.name;
-    if (!name || dds_ret != DDS_RETCODE_OK) {
-      name = "(no name)";
+    std::string name;
+    if (dds_ret == DDS_RETCODE_OK) {
```

Comment:
Nit: enums and litterals on left side of operators

Reply:
Done in 93e4877c3d699d5360ebd9a1889a1d2c3564af1e.

Reply:
Done in 1164ceae56fc7f507ea91c64ee2a4acd8678d592.